extension-androidauto/build.gradle.kts

```diff
@@ -44,7 +44,7 @@ tasks.withType<DokkaTask>().configureEach {
     configureEach {
       reportUndocumented.set(true)
       // https://github.com/mapbox/mapbox-maps-android/issues/301#issuecomment-712736885
```

Comment:
remove this line then? here and everywhere below

Reply:
done.